.github/workflows/test.yaml

```diff
@@ -10,10 +10,11 @@ jobs:
             - uses: shivammathur/setup-php@v2
               with:
                   php-version: '7.4'
+                  tools: cs2pr
             - name: php-cs-fixer
               run: |
                   wget https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/download/v2.18.2/php-cs-fixer.phar -q
-                  php php-cs-fixer.phar fix --dry-run --diff
+                  php php-cs-fixer.phar fix --dry-run --diff --format=checkstyle | cs2pr
```

Comment:
This is nice!

Reply:
@norkunas, you can remove this change - it was done in #240. This will fix the conflicts. We can also use PHP 8 syntax now!

Reply:
Done, but promoted properties are available since PHP8.1 and LiveComponent declares 8.0 support :)

Reply:
Fyi, Php 8.0 does support constructor property promotion: https://www.php.net/releases/8.0/en.php.